website/docs/proof-system/proof-system.md

```diff
@@ -5,68 +5,78 @@ slug: ./
 
 # Proof System Overview
 
+<center> <img src="assets/receipt.png" width="600"/> </center>
+<br/>
```

Comment:
Going by how our STARK By Hand docs work, I think we can do this like
```suggestion
![A diagram of a receipt as the output of a zkVM program](assets/receipt.png)
```

Reply:
Thanks for the suggestion -- the place I'm feeling stuck is when I try to specify a non-default size display...

Reply:
Resizing sounds possible but cumbersome: https://stackoverflow.com/questions/58265585/change-image-size-in-docusaurus

Reply:
Hmm I took a stab at this and it wasn't immediately obvious. I'm going to go ahead and accept the suggestion you proposed and if the sizing looks bad we can fix it on a future PR